.github/workflows/nightly.yml

```diff
@@ -8,6 +8,22 @@ on:
   schedule:
   - cron: '0 3 * * *'
 
+  # TODO remove
+  push:
+    branches:
+      - master
+      - kdsoap-2.1
+      - kdsoap-2.2
+  pull_request:
+    branches:
+      - master
+      - kdsoap-2.1
+      - kdsoap-2.2
+
+env:
+  CLAZY_CHECKS: "level2,no-qstring-allocations,no-fully-qualified-moc-types"
+  CLAZY_IGNORE_DIRS: ".libkode."
```

Comment:
It's a regexp, so it should be ".*libkode.*"
Or maybe it works with a substring, no idea, but at least .libkode. doesn't make sense to me.

Reply:
hmm I think github formatted what you wrote :smile:

Reply:
\.\*libkode\.\*